api-specs/api_spec_PGS.yaml

```diff
@@ -958,4 +987,15 @@ components:
           type: string
           example: "RequestId 77e1c83b-7bb0-437b-bc50-a7a58e5660ac has not been found"
       required:
-        - correlationId
\ No newline at end of file
+        - correlationId
+    CcPaymentInfoResponse:
+      type: object
+      properties:
+        status:
+          type: string
+        responseType:
+          type: string
+        requestId:
+          type: string
+        acsUrl:
+          type: string
```

Comment:
Isn't `status` a required attribute in the response?
Moreover: consider making separate data types for the response (e.g. `CcPaymentInfoAcceptedResponse` with just `status`, `CcPaymentInfoResponse` with everything (also marked required) except `acsUrl`, and `CcPaymentInfoAcsResponse` with every field marked as required).
This would specify the API surface more tightly.